openedx/core/djangoapps/user_api/preferences/api.py

```diff
@@ -113,6 +113,7 @@ def update_user_preferences(requesting_user, update, user=None):
     for preference_key in update.keys():
         preference_value = update[preference_key]
         if preference_value is not None:
+            preference_value = unicode(preference_value)
```

Comment:
These `preference_value` vars now need to be unicode strings in the new DRF version.